password-rotation/README.md

```diff
@@ -0,0 +1,43 @@
+# password-rotation-runner
+
+A Terraform module which deploys a scheduled ECS task on a cron schedule.  This task runs an application which pulls a test user spreadsheet from S3, rotates the Enterprise Portal passwords for the test users, updates the spreadsheet, and pushes it back to S3. To read more about the password rotation application, see the README (TBD)
+
+## Usage
+See [variables.tf](variables.tf) for variable descriptions
+```
+module "password-rotation" {
+  source      = "github.com/CMSgov/portal-test-user-manager//password-rotation
+  
+  app_name    = ""              // defaults to "password-rotation"
+  environment = "dev"
+  task_name   = ""               // defaults to "scheduled-runner"
+
+  repo_url                 = "156322662943.dkr.ecr.us-east-1.amazonaws.com/password-rotation"
+  image_tag                = "" // defaults to "latest"
+  ecs_vpc_id               = "" 
+  ecs_subnet_ids           = [] 
```

Comment:
it could be clearer in this file which fields the user needs to update with their own info, which ones they probably should not change, and which ones have defaults but can be updated